openlr/openlr_model.hpp

```diff
@@ -0,0 +1,107 @@
+#pragma once
+
+#include "geometry/latlon.hpp"
+#include "geometry/point2d.hpp"
+
+#include "std/limits.hpp"
+
+namespace openlr
+{
+// The form of way (FOW) describes the physical road type of a line.
+enum class FormOfAWay
```

Comment:
Can you please clarify? I could not find an object with this name in the spec.
All I found is this:
http://www.openlr.org/maven/map/apidocs/openlr/map/FormOfWay.html
http://www.openlr.org/maven/apidocs/openlr/map/class-use/FormOfWay.html

They use the name I suggested.